fog/ledger/enclave/impl/src/lib.rs

```diff
@@ -191,6 +196,164 @@ where
 
         Ok(())
     }
+
+    fn ledger_store_init(&self, ledger_store_id: ResponderId) -> Result<NonceAuthRequest> {
+        Ok(self.ake.backend_init(ledger_store_id)?)
+    }
+
+    #[allow(unused_variables)]
```

Comment:
nit: Is this needed on this specific line, or can it be moved down to the method (`collate_shard_query_responses`) that has unused variables?

Reply:
Addressed in [278d8c1](https://github.com/mobilecoinfoundation/mobilecoin/pull/2896/commits/278d8c1f98696a964ebd09ae87c4b2d5e316cfd0)